content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,30 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Documentation content standards
+
+Kubernetes documentation does not host certain types of content:
+
+* Vendor, provider, or project-specific content from projects that are not:
+
+    * Inside the kubernetes or kubernetes-sigs GitHub organizations
+    * Inside the CNCF
+
+* Identical content hosted in multiple locations ("dual-sourced" content)
+
+For example:
+
+| Allowed |                           Action                          |                                                    Reason                                                   |
```

Comment:
My gut feeling is that this would be easier to read in the order Action, Allowed, Reason.
@aimeeu to be clear I'm also fine with the current ordering, this feedback wouldn't block me giving an LGTM.

Reply:
expanded